content/docs/command-reference/remote/modify.md

```diff
@@ -464,18 +464,22 @@ a full guide on using Google Drive as DVC remote storage.
   using a custom Google Client project. Also requires using `gdrive_client_id`.
 
   ```dvc
-  $ dvc remote modify myremote \
+  $ dvc remote modify --local myremote \
         gdrive_client_secret 'client-secret'
   ```
 
 - `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
   access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
 
   ```dvc
-  $ dvc remote modify myremote gdrive_user_credentials_file \
-                      .dvc/tmp/myremote-credentials.json
+  $ dvc remote modify --local myremote \
+        gdrive_user_credentials_file .dvc/tmp/mycredentials.json
   ```
 
+> Authentication values may contain sensitive user info. Therefore, it's safer
```

Comment:
what values does it apply to now? (jus to check that we have something)

Reply:
gdrive_client_secret and gdrive_user_credentials_file

Reply:
they are both not sensitive